.github/workflows/Test.yml

```diff
@@ -8,14 +8,10 @@ jobs:
     - uses: actions/checkout@v2
       with:
         submodules: 'recursive'
-    - name: Setup .NET Core
+    - name: Setup .NET 6
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: '3.1.x'
-    - name: Setup .NET 5
-      uses: actions/setup-dotnet@v1
-      with:
-        dotnet-version: '5.0.100-preview.8.20417.9'
+        dotnet-version: '6.0.201'
```

Comment:
Usually better to not fix the last version, `6.0.x`

Reply:
Nah. I've had enough fuckery with non-determinstic/irreproducable builds (exactly when instead of fixing the version, I opened it)

Reply:
ok